apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1962,6 +2152,30 @@ slash_tx(Account, Nonce, _Round, SignedTx, #data{ on_chain_id = ChanId
                                   , nonce      => Nonce }),
     {ok, Tx, []}.
 
+settle_tx_for_signing(#data{} = D) ->
+    Account = my_account(D),
+    {ok, Nonce} = aec_next_nonce:pick_for_account(Account),
+    settle_tx(Account, Nonce, D).
+
+settle_tx(Account, Nonce, #data{ opts  = #{initiator := I,
+                                           responder := R} = Opts
+                               , state = State } = D) ->
```

Comment:
I see the settle tx depends heavily on the latest state. Although this could be fragile with the play of Force Progress on-chain.
This is outside of the scope of this PR, in order for us not to lose it - tracked as [SC: Make settling force-progress aware](https://www.pivotaltracker.com/story/show/165977998)

Reply:
As same reasoning applies to solo close as well, I extended the PT